app/webpacker/src/stylesheets/_commodity-tree.scss

```diff
@@ -554,3 +555,8 @@ article.tariff {
     }
   }
 }
+
+.additional-code-table {
+  font-size: 14px;
```

Comment:
rather then duplicate govuk sizes, we should defer to them 
- see https://github.com/trade-tariff/trade-tariff-frontend/blob/main/app/webpacker/src/stylesheets/_commodity-codes.scss#L89

And for the margins
- https://design-system.service.gov.uk/styles/spacing/#applying-spacing-in-your-own-css